pulsar-client/src/main/java/org/apache/pulsar/client/api/PulsarClient.java

```diff
@@ -122,6 +122,21 @@ public static PulsarClient create(String serviceUrl, ClientConfiguration conf) t
      */
     Consumer subscribe(String topic, String subscription) throws PulsarClientException;
 
+    /**
+     * Subscribe to the given topic and subscription combination with default {@code ConsumerConfiguration}
+     *
+     * @param topic
+     *            The name of the topic
+     * @param subscription
+     *            The name of the subscription
+     * @param listener
+     *            A listener that will be called in order for every message received
+     * @return The {@code Consumer} object
+     * @throws PulsarClientException
+     * @throws InterruptedException
+     */
+    Consumer subscribe(String topic, String subscription, MessageListener listener) throws PulsarClientException;
```

Comment:
@mgodave  Can you mark the new methods with `@since 1.22`? 
The reason is that if we end up adding the builder pattern for creating a consumer before the next release is out, it would be easy to just remove these methods instead of deprecating them.